src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommandParser.cs

```diff
@@ -9,18 +9,15 @@ namespace Microsoft.DotNet.Cli
     internal static class WorkloadListCommandParser
     {
         // arguments are a list of workload to be detected
-        public static readonly Option MachineReadableOption = new Option<bool>("--machine-readable")
-        {
-            IsHidden = true
-        };
-
+        public static readonly Option MachineReadableOption = new Option<bool>("--machine-readable") {IsHidden = true};
         public static readonly Option VerbosityOption = CommonOptions.VerbosityOption();
 
         public static Command GetCommand()
         {
             var command = new Command("list", LocalizableStrings.CommandDescription);
             command.AddOption(MachineReadableOption);
             command.AddOption(VerbosityOption);
+            command.AddOption(WorkloadUpdateCommandParser.SdkVersionOption);
```

Comment:
nit: if we want to have separate option descriptions for this option when applied to list vs update we'll have to have a separate copy here for list.

Reply:
This is for internal use, so we don't need to have a too detailed description